### PR TITLE
Newsletter-133:Translate into Japanese

### PR DESCRIPTION
**Pull request by azuchi**

### CODE REVIEW

File: _posts/ja/newsletters/2021-01-27-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  軽量クライアントには未承認トランザクションが有効か検証する手段がないため、圧縮されたブロックデータを持つ軽量クライアントにのみサービスを提供すると説明しています。
+
+## リリースとリリース候補
+
+*人気のBitcoinインフラストラクチャプロジェクトの新しいリリースとリリース候補。
+新しいリリースにアップグレードしたり、リリース候補のテストを支援することを検討してください。*
+
+- [C-Lightning 0.9.3][c-lightning 0.9.3]はプロジェクトの最新のマイナーリリースです。
+  これには、ユーザーインターフェースとプラグインの機能に加えて、提案されているOnion messageプロトコル（[Newsletter #92][news92 cl3600]参照）
+  とOfferプロトコル（[Newsletter #128][news128 cl4255]参照）の実験的なサポートが含まれています。詳細は[変更履歴][cl cl]を参照してください。
```
Comment (AkioNak) [inline _posts/ja/newsletters/2021-01-27-newsletter.md:61]: s/詳細は/詳細は[リリースノート][c-lightning 0.9.3]と/

File: _posts/ja/newsletters/2021-01-27-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  Bitcoin CoreでのUSDTプローブの潜在的なユースケースを探るため、課題[#20981][Bitcoin Core #20981]が公開されました。
+
+- [Bitcoin Core #17920][]では 、macOS用のBitcoin Coreのバイナリに
+  GNU Guixを使った[再現性のあるビルド][topic reproducible builds]のためのサポートを追加します。
+  WindowsといくつかのLinuxプラットフォームはすでにサポートされているため、新しいGuix決定性ビルドシステムは
+  既存のGitianシステムと同じプラットフォームをすべてサポートするようになりました。
+
+- [LND #4908][]では、[anchor output][topic anchor outputs]を使っているチャネルが、
+  場合によって確保していた残高を適用することでチャネル閉鎖時にコミットメントトランザクションの手数料をバンプできることを保証します。
+  とりわけ、通常の`SendCoins` RPC呼び出しでは、`send_all`がセットされている場合を除いて、この確保していた残高はまだ適用されません。
```
Comment (AkioNak) [inline _posts/ja/newsletters/2021-01-27-newsletter.md:87]: In this case, I think 'とはいえ' is better than ' とりわけ'.

**Comment by azuchi** [comment]
Thanks! @AkioNak. I fixed.

**Comment by AkioNak** [comment]
It looks good to me. Thanks @azuchi .

**Review by bitschmidty — COMMENTED** [review]
ACK

- Built and smoke tested links

**Pull request merged**